AzureResourceManager/README.md

```diff
@@ -0,0 +1,80 @@
+# Setup Artifactory Enterprise
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FJFrogDev%2FJFrog-Cloud-Installers%2Fmaster%2FAzureResourceManager%2Fazuredeploy.json" target="_blank">
+<img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FJFrogDev%2FJFrog-Cloud-Installers%2Fmaster%2FAzureResourceManager%2Fazuredeploy.json" target="_blank">
+<img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template can help you setup the [Artifactory Enterprise](https://jfrog.com/artifactory) on Azure.
+
+## A. Deploy Artifactory Enterprise on Azure
+![screenshot](images/HA_Diagram.png)
+
+
+1. Click "Deploy to Azure" button. If you haven't got an Azure subscription, it will guide you on how to signup for a free trial.
+
+2. Enter a valid values to parameters. Atleast 1 license has to be provided.
+
+
+![screenshot](images/Parameters.png)
+
+3. Click on Purchase to start deploying resources. It will deploy MsSQL database, Azure Blob storage container, VM installing Nginx and Artifactory and Load balancer.
+
+4. Once deployment it done. Copy FQDN from Output of deployment template.
+
+5. Access artifactory using FQDN. 
+
+6. You will see specified artifactory member nodes in 'Admin ->  High Availability' page.
+
+### Note: 
+1. This template only supports Artifactory version 5.8.x and above.
+2. Turn off daily backups.  Read Documentation provided [here](https://www.jfrog.com/confluence/display/RTF/Managing+Backups)

```

Comment:
If there's no easy way to do this by default on load, we should probably reach out to R&D to file that as a feature request.